lib/index-api.js

```diff
@@ -33,6 +33,16 @@ module.exports = function(config, auth, storage) {
   app.use(expressJson5({ strict: false, limit: config.max_body_size || '10mb' }))
   app.use(Middleware.anti_loop(config))
 
+  // encode / in a scoped package name to be matched as a single parameter in routes
+  // TODO: routes should be modified to properly handle scopes, this is just a workaround
```

Comment:
As far as I understand it, it's a workaround for a nginx bug, and sinopia already handles scopes correctly.

This workaround is still useful to have though.